provisioning/miniprov/create-template.sh

```diff
@@ -23,9 +23,9 @@ FILENAME=$1
 URL="json://$(readlink -f "$FILENAME")"
 
 conf "$URL" set 'cluster>my-cluster>site>storage_set_count=1'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>durability>data=1'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>durability>parity=0'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>durability>spare=0'
+conf "$URL" set 'cluster>my-cluster>storage_set[0]>durability>sns>data=4'
```

Comment:
With this change, is 1-node provisioner going to work in the same way as it is working currently and these values will get reflected only for 3-node?

Reply:
This is discussed during 1:1 phone call with Suvrat. 
In short, 
1. This file is not used by any external team
2. It is used from time to time to re-generate some template files.
So it is not a problem that this file has some discrepancies. At the times when it was created there were no separate 3-node and 1-node templates. In order to address the issue properly, we need to understand whether 4,2,2 numbers should be some VAR_TMPL placeholders (given it is a template). Anyway it is not a must for this PR.